errata_tool/cdn_repo.py

```diff
@@ -0,0 +1,96 @@
+from errata_tool import ErrataConnector
+
+
+class CdnRepo(ErrataConnector):
+    """A CdnRepo contains a dict, for example:
+    {
+        'id': 12189,
+        'relationships': {
+            'arch': {
+                'id': 28,
+                'name': 'multi'
+            },
+            'packages': [
+                {
+                    'cdn_repo_package_tags': [
+                        {
+                            'id': 16054,
+                            'tag_template': '{{version}}-{{release}}'
+                        }, {
+                            'id': 16055,
+                            'tag_template': '{{version}}'
+                        }, {
+                            'id': 19195,
+                            'tag_template': 'v{{version(2)}}'
+                        }
+                    ],
+                    'id': 49826,
+                    'name': 'work-container'
+                }
+            ],
+            'variants': [
+                {
+                    'id': 3072,
+                    'name': '8Base-RHACM-2.0'
+                }, {
+                    'id': 3159,
+                    'name': '8Base-RHACM-2.1'
+                }, {
+                    'id': 3284,
+                    'name': '8Base-RHACM-2.2'
+                }, {
+                    'id': 3421,
+                    'name': '8Base-RHACM-2.3'
+                }
+            ]
+        },
+        'type': 'cdn_repos'
+    }
+    """
+
+    def __init__(self, name, product_version, data=None):
+        self.name = name
+        self.data = data
+        self.product_version = product_version
+        self.url = self._url + '/product_versions/%s/cdn_repos/%s' % \
+            (self.product_version, self.name)
+
+    def refresh(self):
+        url = '/api/v1/cdn_repos?filter[variant_name]=%s' % self.name
```

Comment:
`filter[variant_name]` -> `filter[name]`

Reply:
Perhaps `'/api/v1/cdn_repos?filter[variant_name]=%s'` => `/api/v1/cdn_repos/%s` ?

Reply:
Oh, yeah, CLOUDWF-316 is resolved so this is even better.

Reply:
Great!